### PR TITLE
test case for page meta

### PR DESCRIPTION
**Pull request by bsfsnehal**
### Description

test case for page meta
### How has this been tested?

Branch name :- page-meta
Execution :- npm run test:e2e:interactive — tests/e2e/specs/customizer/page-meta/page-meta.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/page-meta/page-meta.test.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { wpDataSelect } from '@wordpress/e2e-test-utils';
+import { setCustomize } from '../../../utils/set-customize';
+import {createURL, createNewPost, publishPost,} from '@wordpress/e2e-test-utils';
```
Comment (sushmak02) [inline tests/e2e/specs/customizer/page-meta/page-meta.test.js:3]: comma after publishPost {}

File: tests/e2e/specs/customizer/page-meta/page-meta.test.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        const pageSidebar = {
+            'single-page-sidebar-layout': 'left-sidebar',
+        };
+        await setCustomize( pageSidebar );
+        await createNewPost( {
+            postType: 'page',
+            title: 'Test-page',
+        } );
+        await publishPost();
+        await page.goto( createURL( 'Test-page' ), {
```
Comment (sushmak02) [inline tests/e2e/specs/customizer/page-meta/page-meta.test.js:15]: When accessing any page URL WordPress do not support capital letter in URL. 
Instead on Test-page -> test-page is expected

File: tests/e2e/specs/customizer/post-meta/post-meta.test.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import { wpDataSelect } from '@wordpress/e2e-test-utils';
+import { setCustomize } from '../../../utils/set-customize';
+import {createURL, createNewPost, publishPost,} from '@wordpress/e2e-test-utils';
+describe( 'Post meta setting for sidebar', () => {
+	it( 'position of Sidebar for page as LEFT should apply correctly', async () => {
+        const postSidebar = {
+            'single-post-sidebar-layout': 'left-sidebar',
+            'site-sidebar-layout': ' Right Sidebar',
```
Comment (sushmak02) [inline tests/e2e/specs/customizer/post-meta/post-meta.test.js:8]: Unused variable and its value is also not valid.

**Comment by bsfsnehal** [comment]
Closing PR

**Comment by sushmak02** [comment]
This PR was for demo purposes. Closing the PR.

**Pull request closed without merging**